src/analytics/config.py

```
@@ -74,6 +74,15 @@ class BaseConfig:
     BIGQUERY_COHORTS = env_var("BIGQUERY_COHORTS")
     DEVICES_SUMMARY_TABLE = env_var("DEVICES_SUMMARY_TABLE")
 
+    extra_time_grouping = {"daily", "weekly", "monthly", "yearly"}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)